status-reporter/status_reporter/manager_reporter.py

```diff
@@ -0,0 +1,26 @@
+#########
+# Copyright (c) 2019 Cloudify Platform Ltd. All rights reserved
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#       http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+#  * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#  * See the License for the specific language governing permissions and
+#  * limitations under the License.
+
+from .constants import ComponentType
+from .status_reporter import Reporter
+
+
+def collect_status():
+    pass
+
+
+def main():
+    reporter = Reporter(collect_status, ComponentType.MANAGER)
```

Comment:
What's the point of this one if it's doing nothing? Wouldn't we be better just not having it?